CONTRIBUTING.rst

```diff
@@ -36,3 +36,15 @@ Look in ``.github/workflows/`` to find the commands.
 For example, to run tests::
 
     $ python3 -m pytest
+
+Possible future changes
+-----------------------
+
+``flake8-pyi`` aims to provide support for modern conventions in writing
+typed Python code, such as using ``|`` instead of ``Union`` and using the
+``list`` builtin instead of ``typing.List``.
```

Comment:
How about this for the first one? I think it maybe makes it a little clearer that this is about the broad direction of travel, rather than an actionable item

```suggestion
Development roadmap
-------------------------

As ``flake8-pyi`` continues to develop, our priorities include providing
further support for modern conventions in writing typed Python code,
such as using ``|`` instead of ``Union`` and using the ``list`` builtin
instead of ``typing.List``.
```

Reply:
This still feels like it should be an issue to me. I created #80.